docs/docs/reference/functions/middleware-and-helpers.md

```diff
@@ -51,7 +51,41 @@ export default async function corsHandler(req, res) {
 
 ## Custom body parsing
 
-Generally useful for file upload support.
+Generally useful for file upload support or raw bodies.
 
-This is not yet supported. [Add a comment in the discussion if this is an
-important use case for you](https://github.com/gatsbyjs/gatsby/discussions/30735).
+Each function can export an object named `config`. This object will allow you to control the `body-parser` middleware used by Gatsby Functions. The `limit` property will allow configuration of payload size up to **_32mb_**.
```

Comment:
Is there an API reference we can link to for `body-parser`? The examples don't make it clear if this is all the options one have